Formula/openjdk.rb

```diff
@@ -22,79 +20,117 @@ class Openjdk < Formula
     depends_on "alsa-lib"
   end
 
-  # From https://jdk.java.net/archive/
+  # From https://jdk.java.net/
+  # http://openjdk.java.net/groups/build/doc/building.html#boot-jdk-requirements
+  # TODO: Determine if N-1 is advised in a future release
+  #   e.g. "configure: (Your Build JDK must be version 16)"
   resource "boot-jdk" do
     on_macos do
-      url "https://download.java.net/java/GA/jdk14.0.2/205943a0976c4ed48cb16f1043c5c647/12/GPL/openjdk-14.0.2_osx-x64_bin.tar.gz"
-      sha256 "386a96eeef63bf94b450809d69ceaa1c9e32a97230e0a120c1b41786b743ae84"
+      url "https://download.java.net/java/early_access/jdk16/26/GPL/openjdk-16-ea+26_osx-x64_bin.tar.gz"
+      sha256 "85a6bb4f73c1b7ab64cb044b368581d0c72eefed2142a06b143f62f0e4325353"
     end
     on_linux do
-      url "https://download.java.net/java/GA/jdk14.0.2/205943a0976c4ed48cb16f1043c5c647/12/GPL/openjdk-14.0.2_linux-x64_bin.tar.gz"
-      sha256 "91310200f072045dc6cef2c8c23e7e6387b37c46e9de49623ce0fa461a24623d"
+      url "https://download.java.net/java/early_access/jdk16/26/GPL/openjdk-16-ea+26_linux-x64_bin.tar.gz"
+      sha256 "e74300f3f770122358d768d45e181d29c710e8d41bbc4ab8e492929f2867347c"
     end
   end
 
-  # Fix build on Xcode 12
-  # https://bugs.openjdk.java.net/browse/JDK-8253375
-  patch do
-    url "https://github.com/openjdk/jdk/commit/f80a6066e45c3d53a61715abfe71abc3b2e162a1.patch?full_index=1"
-    sha256 "5320e5e8db5f94432925d7c240f41c12b10ff9a0afc2f7a8ab0728a114c43cdb"
-  end
-
-  # Fix build on Xcode 12
-  # https://bugs.openjdk.java.net/browse/JDK-8253791
-  patch do
-    url "https://github.com/openjdk/jdk/commit/4622a18a72c30c4fc72c166bee7de42903e1d036.patch?full_index=1"
-    sha256 "4e4448a5bf68843c21bf96f510ea270aa795c5fac41fd9088f716822788d0f57"
+  # Calculate Xcode's dual-arch JavaNativeFoundation.framework path
+  def framework_path
+    File.expand_path("../SharedFrameworks/ContentDeliveryServices.framework/Versions/Current/itms/java/Frameworks",
+      MacOS::Xcode.prefix)
   end
 
   def install
     boot_jdk_dir = Pathname.pwd/"boot-jdk"
     resource("boot-jdk").stage boot_jdk_dir
-    boot_jdk = boot_jdk_dir/"Contents/Home"
+    boot_jdk = boot_jdk_dir.to_s
```

Comment:
```suggestion
    boot_jdk = boot_jdk_dir
```

Reply:
Marking as resolved due to lower comment by project maintainer.  I hope that's ok. :)